app/playground/client-error/page.tsx

```diff
@@ -0,0 +1,8 @@
+import PageClient from './page.client';
+
+// Avoid statically building this page during build.
+export const dynamic = 'force-dynamic';
+
+export default async function Page() {
+  return <PageClient />;
+}
```

Comment:
Not sure if this is important, but when i navigate to `http://localhost:3000/playground/client-error` there's a re-render issue causing over 200 requests.